.gitignore

```diff
@@ -0,0 +1,39 @@
+HELP.md
```

Comment:
Why?

Reply:
It's from default spring boot project from intellij idea. Fixed in #5 PR